_data/sidebars/scos_dev_sidebar.yml

```diff
@@ -4717,8 +4717,6 @@ entries:
       nested:
         - title: Upgrader tool overview
           url: /docs/scos/dev/guidelines/keeping-a-project-upgradable/upgrader-tool-overview.html
```

Comment:
The article is moved and renamed. Please update `url` and `title`

Reply:
I used the master branch and didn't change it, so why should I update something here? I don't get it